src/clr-angular/data/datagrid/built-in/filters/datagrid-property-numeric-filter.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
```

Comment:
Noticed we'll need all the copyrights updated for these files to 2019. Current master has a script to do this for you though, so you could wait until rebasing.

Reply:
Cool, holding off for now.